packages/stryker/src/SourceFile.ts

```diff
@@ -107,7 +107,9 @@ export default class SourceFile {
           if (this.file.textContent.charCodeAt(pos) === CharacterCodes.lineFeed) {
             pos++;
           }
-        // falls through
+          result.push(lineStart);
```

Comment:
Could we extract these 2 lines into a method? Something like `markLineStart`? It's fine if the function is local to `computeLineStarts`. That way, you don't have to pass along the variables.

Reply:
Sure! :+1: